### DISCUSSION TITLE
A Better Description of How to Import R Generated Loom File without Splicing Info to Scvelo

### DISCUSSION THREAD

**Discussion by paulitikka** [theislab/scvelo#745] [Q&A]
Hi,

It would be nice if you could tell how to import R Generated Loom File without splicing info to scVelo?
Perhaps once can just combine the loom file to splice data, which I have, but how? Do I need to have velocyto.R installed?
I have experienced issues to install velocyto both to R and python and unix. Is it not possible to get spliced data without velocyto.R? If this importing issue has been already solved somewhere, please just provide me the link this time. Thanks!

Best,
Pauli

Fyi for reference:
First I convert my big scRNAseq (Seurat) data to loom in R:
DefaultAssay(big) <- "RNA"
SaveLoom(big, filename="bota.loom")

And this is how I import the file in python :
adatan=scanpy.read_loom("bota.loom")

#If I run it without splicing data this is what I get:
#runcell('The moments for normal and stochastic velocity analyese:', 'D:/Codes and Instructions/scVelo real data_tikka11121.py')
#WARNING: Could not find spliced / unspliced counts.
#Traceback (most recent call last): #...
#KeyError: 'spliced'

**Comment by vondoRishi** [comment]
I have not understood completely what is your objective. 
However, in my project, I have followed

- To generate loom files these [instructions](https://velocyto.org/velocyto.py/tutorial/index.html#running-the-cli)

- ITo merge with seurat objects these [instructions](https://github.com/velocyto-team/velocyto.R/issues/16#issuecomment-461067873) in my project.

**Comment by paulitikka** [comment]
Thank you for your links. The main question is how to convert bam files to spliced/unspliced data needed for scvelo.

I found a possible [solution ](https://combine-lab.github.io/alevin-tutorial/2020/alevin-velocity/), but got stuck to a place where one would need to install latest salmon. Installing salmon has been also found to difficult [elsewhere/someone else](https://www.biostars.org/p/378380/).
If you could help me installing salmon program, I would greatly appreciate.
